big_tests/tests/last_SUITE.erl

```diff
@@ -121,6 +128,25 @@ last_server(Config) ->
                           true = (get_last_activity(Stanza) > 0)
                   end).
 
+user_not_subscribed_receives_error(Config) ->
+    escalus:fresh_story(Config, [{alice, 1}, {bob, 1}], fun(Alice, Bob) ->
+        %% Alice asks about Bob's last activity
+        BobShortJID = escalus_client:short_jid(Bob),
+        escalus_client:send(Alice, escalus_stanza:last_activity(BobShortJID)),
+
+        %% server replies with an error, since there is no subscription
+        Stanza = escalus_client:wait_for_stanza(Alice),
+        escalus:assert(is_iq_error, Stanza),
```

Comment:
I think it'd be good to check what is the error. The assertion checks only if the iq stanza is of type error. Inside there can be sth else then we expect.